realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -66,6 +67,7 @@
     RealmSchema schema;
     Handler handler;
     HandlerController handlerController;
+    List<Runnable> asyncTransactionCallbacks = new ArrayList<Runnable>();
```

Comment:
`RealmThreadPoolExecutor.transactions` is for warnings and `BaseRealm.asyncTransactionCallbacks ` is for pending transactions. Although they can be moved into other classes such as `Realm` or `HandlerController`, this kind of list is still needed.